moveit_setup_assistant/CMakeLists.txt

```diff
@@ -27,7 +27,7 @@ find_package(catkin REQUIRED COMPONENTS
 include_directories(${catkin_INCLUDE_DIRS})
 
 # Qt Stuff
-if(rviz_QT_VERSION VERSION_LESS "5")
+if("${rviz_QT_VERSION}" VERSION_LESS "5")
```

Comment:
In my opinion, this is a regression. Using a stringified version of the variable, you will not notice typos.

Reply:
@rhaschke Please fight this out with the `catkin_lint` maintainer if your passionate about it.
I see your point, but the tool currently states:

```
moveit_setup_assistant: CMakeLists.txt(32): notice: operands for operator VERSION_LESS should be quoted strings
```
I believe they never considered users might use variable names here. (They should...)

As long as it's not changed there, I would prefer to get rid of the lint notice.

Reply:
@v4hn I added an appropriate issue: https://github.com/fkie/catkin_lint/issues/61
I'm not very passionate about this, but we shouldn't follow such hints if they decrease code readability. `catkin_lint` reports this as a notice only, which is hidden if you use the default warning level.